src/TaskQueue.js

```diff
@@ -0,0 +1,44 @@
+const iterate = require('taskcluster-lib-iterate');
+const assert = require('assert');
+class TaskQueue {
+  constructor(cfg, queue) {
+    assert(cfg.worker.workerId, 'Worker ID is required');
+    assert(cfg.worker.workerType, 'Worker type is required');
+    assert(cfg.worker.workerGroup, 'Worker group is required');
+    assert(cfg.worker.provisionerId, 'Provisioner ID is required');
+    assert(queue, 'Instance of taskcluster queue is required');
+    this.queue = queue;
+    this.workerType = cfg.worker.workerType;
+    this.provisionerId = cfg.worker.provisionerId;
+    this.workerGroup = cfg.worker.workerGroup;
+    this.workerId = cfg.worker.workerId;
+  }
+  async startWorker() {
+
```

Comment:
Fix the whitespace here: a blank line between each function, and no blank line between the declaration of the function and the first line.  So basically move this blank line up one line :)